meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend

```diff
@@ -80,4 +80,4 @@ RDEPENDS:${PN}-lite = "\
     ${@bb.utils.contains('PACKAGECONFIG', 'aklite-offline', '${PN}-lite-offline', '', d)} \
 "
 RDEPENDS:${PN}-lite-lib = "${RDEPENDS:aktualizr}"
-RDEPENDS:${PN}-lite-apps = "docker-compose skopeo"
\ No newline at end of file
+RDEPENDS:${PN}-lite-apps = "docker-compose skopeo"
```

Comment:
Is this added mistakenly? If not, then maybe should be in dedicated commit.

Reply:
Yeah, docker-compose is new here.

Reply:
Will merge anyway as this should change the behavior, but better to separate next time.